README.md

```diff
@@ -102,6 +103,23 @@ Succinct help about any command or subcommand is available via the `-h, --help`
 flag. Verbose help about any command or subcommand is available via the help
 argument, e.g. `fastly help service`.
 
+## Bash/ZSH Shell Completion
+The CLI can generate completions for all commands, subcommands and flags.
+
+By specifying `--completion-bash` as the first argument, the CLI will show possible subcommands. By ending your argv with `--`, hints for flags will be shown.
+
+### Installation by bash_profile
```

Comment:
Since "bash" is not inclusive language 😜 perhaps this section should be called "Configuring your shell" (or similar)